fluid/fluid.cpp

```diff
@@ -335,7 +334,7 @@ void Fluid::program_change(int chan, int prognum)
 
       Preset* preset = find_preset(banknum, prognum);
       if (!preset) {
-            qDebug("Fluid::program_change: preset %d %d not found", banknum, prognum);
+            //qDebug("Fluid::program_change: preset %d %d not found", banknum, prognum);
```

Comment:
these are just going to produce tons of qDebug messages when playing back without a soundfont, which will slow down playback, so I removed them.